src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.ProjectFormat/TryConvertTool.cs

```diff
@@ -54,6 +57,11 @@ public class TryConvertTool : ITryConvertTool
         public string GetCommandLine(IProject project)
             => Invariant($"{Path} {GetArguments(project.Required())}");
 
+        private string? GetMSBuildPath()
+        {
+            return _locator.MSBuildPath?.Replace("\\", "\\\\");
```

Comment:
I feel like I'm still missing something here. Can you give an example of a scenario that won't work without the `.Replace` to help me get my head around it?

Reply:
I think it's because we're passing args to the process as a single string rather than an array of strings. but when i run without the escapes I see that try-convert receives all arguments after the msBuildPath as part of the path

![image](https://user-images.githubusercontent.com/23503973/116758877-ab7b1d80-a9d6-11eb-8633-dca302409d45.png)